src/components/Card/Card.jsx

```diff
@@ -1,7 +1,53 @@
 import React from 'react';
+import theme from 'styles/theme';
+import styled from 'styled-components';
+import { useState, useEffect } from 'react/cjs/react.development';
 
-function Card(props) {
-  return <div>카아-드 컴포넌트다</div>;
+function Card({ id, username, top_detail, bottom_detail, img }) {
+  return (
+    <Cards>
+      <TempDatas>
+        <TempDataTitle>{username}</TempDataTitle>
+        <ImgBoxs>
+          <img src={img} />
+        </ImgBoxs>
+        <BottomDetailTemp>{bottom_detail}</BottomDetailTemp>
+        <TopDetailTemp>{top_detail}</TopDetailTemp>
+      </TempDatas>
+    </Cards>
+  );
 }
 
+const Cards = styled.div`
+  display: flex;
+  max-width: 1140px;
+  margin: 0 auto;
+  justify-content: space-evenly;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+const TempDatas = styled.p`
+  border: 1px solid gray;
+  width: 250px;
```

Comment:
```suggestion
  flex-flow: column wrap;
`;

const TempDatas = styled.p`
  border: 1px solid gray;
  width: 250px;
```
구분되는 요소들은 가독성을 위해 한줄씩 다 띄워주세요